packages/review/testing/src/config/testConfigOperator.ts

```diff
@@ -31,22 +31,19 @@ class TestConfigOperator {
     return this._testConfig;
   }
 
-  public mergeJestConfig(commingConfig: JestConfig) {
-    this._jestConfig = merge({}, this._jestConfig, commingConfig);
+  public mergeJestConfig(commonConfig: JestConfig) {
```

Comment:
I think `commingConfig` is not a typo, it means `newConfig`

Reply:
So it's actually `comingConfig`.

But I still find the word weird, not even `newConfig` is appropriate, maybe we should use `userConfig`?

Reply:
After thinking about it, if I were to design a `merge`, I would call the incoming new parameter `source` and the original parameter `target`.

Reply:
`source` and `target` is fine

Reply:
OK, I change `commonConfig` to `sourceConfig`.